name: Supervised Classification, features: MiceProtein
{'X_train': {'cols': 0.0156948675467194, 'rows': 0.47050754458161864}, 'X_valid': {'cols': 0.013767653111915409, 'rows': 0.43169398907103823}, 'X_test': {'cols': 0.018518518518518517, 'rows': 0.5222222222222223}}
dataset sizes
(808, 77) (183, 77) (270, 77)
dropped dataset sizes
(430, 77) (104, 77) (129, 77)
key: 7960, k: 1/4, dataset: 40966, missing: None, impute: None
training gbm for 5000 epochs
{'num_leaves': 19, 'max_bin': 53, 'max_depth': 6, 'min_data_in_leaf': 11, 'learning_rate': 0.07955865781976543, 'num_iterations': 448, 'objective': 'softmax', 'verbose': -1, 'num_class': 8}
Training until validation scores don't improve for 100 rounds
Early stopping, best iteration is:
[68]	valid_0's multi_logloss: 0.0946467
strategy:None, acc gbm: 0.9740740740740741
strategy:None, nll xbg:0.13691750168800354
name: Supervised Classification, features: MiceProtein
{'X_train': {'cols': 0.017486738613499177, 'rows': 0.4943661971830986}, 'X_valid': {'cols': 0.014227345688019845, 'rows': 0.43820224719101125}, 'X_test': {'cols': 0.014718614718614718, 'rows': 0.48148148148148145}}
dataset sizes
(744, 77) (178, 77) (270, 77)
dropped dataset sizes
(376, 77) (100, 77) (140, 77)
key: 4255, k: 2/4, dataset: 40966, missing: None, impute: None
training gbm for 5000 epochs
{'num_leaves': 19, 'max_bin': 53, 'max_depth': 6, 'min_data_in_leaf': 11, 'learning_rate': 0.07955865781976543, 'num_iterations': 448, 'objective': 'softmax', 'verbose': -1, 'num_class': 8}
Training until validation scores don't improve for 100 rounds
/home/jahan/.conda/envs/jax/lib/python3.10/site-packages/lightgbm/engine.py:177: UserWarning: Found `num_iterations` in params. Will use it instead of argument
  _log_warning(f"Found `{alias}` in params. Will use it instead of argument")
/home/jahan/.conda/envs/jax/lib/python3.10/site-packages/lightgbm/engine.py:177: UserWarning: Found `num_iterations` in params. Will use it instead of argument
  _log_warning(f"Found `{alias}` in params. Will use it instead of argument")
Did not meet early stopping. Best iteration is:
[385]	valid_0's multi_logloss: 0.0184062
strategy:None, acc gbm: 0.9925925925925926
strategy:None, nll xbg:0.03292205184698105
name: Supervised Classification, features: MiceProtein
{'X_train': {'cols': 0.017215342796738146, 'rows': 0.48320413436692505}, 'X_valid': {'cols': 0.013522559914312492, 'rows': 0.4329896907216495}, 'X_test': {'cols': 0.017075517075517074, 'rows': 0.4703703703703704}}
dataset sizes
(808, 77) (194, 77) (270, 77)
dropped dataset sizes
(414, 77) (110, 77) (143, 77)
key: 9235, k: 3/4, dataset: 40966, missing: None, impute: None
training gbm for 5000 epochs
{'num_leaves': 19, 'max_bin': 53, 'max_depth': 6, 'min_data_in_leaf': 11, 'learning_rate': 0.07955865781976543, 'num_iterations': 448, 'objective': 'softmax', 'verbose': -1, 'num_class': 8}
Training until validation scores don't improve for 100 rounds
Did not meet early stopping. Best iteration is:
[423]	valid_0's multi_logloss: 0.0184227
strategy:None, acc gbm: 0.9925925925925926
strategy:None, nll xbg:0.02302476391196251
name: Supervised Classification, features: MiceProtein
{'X_train': {'cols': 0.016410220214568044, 'rows': 0.47690217391304346}, 'X_valid': {'cols': 0.015316205533596838, 'rows': 0.483695652173913}, 'X_test': {'cols': 0.01683501683501683, 'rows': 0.48148148148148145}}
dataset sizes
(760, 77) (184, 77) (270, 77)
dropped dataset sizes
(397, 77) (95, 77) (140, 77)
key: 8274, k: 4/4, dataset: 40966, missing: None, impute: None
training gbm for 5000 epochs
{'num_leaves': 19, 'max_bin': 53, 'max_depth': 6, 'min_data_in_leaf': 11, 'learning_rate': 0.07955865781976543, 'num_iterations': 448, 'objective': 'softmax', 'verbose': -1, 'num_class': 8}
Training until validation scores don't improve for 100 rounds
/home/jahan/.conda/envs/jax/lib/python3.10/site-packages/lightgbm/engine.py:177: UserWarning: Found `num_iterations` in params. Will use it instead of argument
  _log_warning(f"Found `{alias}` in params. Will use it instead of argument")
/home/jahan/.conda/envs/jax/lib/python3.10/site-packages/lightgbm/engine.py:177: UserWarning: Found `num_iterations` in params. Will use it instead of argument
  _log_warning(f"Found `{alias}` in params. Will use it instead of argument")
Did not meet early stopping. Best iteration is:
[360]	valid_0's multi_logloss: 0.0720235
strategy:None, acc gbm: 0.9629629629629629
strategy:None, nll xbg:0.10923141986131668